app/presenters/cohorts/students_presenter.rb

```diff
@@ -14,8 +14,8 @@ def t(key, variables = {})
     def filter
       @filter ||=
         begin
-          milestone_targets_data =
-            milestone_targets
+          milestones_data =
+            milestones
```

Comment:
This wasn't a good rename. This scope isn't actually assignments - these are still targets. A better name could be `targets_with_milestone` or something similar. The same applies to a few other renames in this class.